packages/outputarea/src/widget.ts

```diff
@@ -412,15 +439,76 @@ export class OutputArea extends Widget {
   /**
    * Render and insert a single output into the layout.
    */
```

Comment:
Would it be worth adding some documentation now that we are updating the method?

For all the params?
```
    index: number,
    model: IOutputModel,
    force?: boolean
```

Reply:
Done. I have anyway removed the force optional param and reverted back to the previous signature.